### ISSUE TITLE
Reload Current Page UI

### ISSUE THREAD

**Issue by lasers** [gorhill/uBlock#2201]
Hi there. I noticed a possible minor bug that UBO does not detect or cannot know that I hadn't refreshed yet. I don't know if it's worth fixing as couple of reloads will fix it... but here it is.

* Visit the toolbar icon.
* Hit that big pause button.
* `Reload Current Page` button appears.
* Click somewhere else. Make it go away.
* Hit that big pause button again.
* No `Reload Current Page` button.
* Not able to reload. Do it via builtin browser.

The Reload button might not want to go away if the page hadn't been refreshed yet.

EDIT: Another workaround... Instead of loading pages, you can keep clicking on the Toolbar Icon to get that off/on effect... to get the one you want... and Reload button will appear.

FF 50 / UBO 1.10.0

**Comment by gorhill** [comment]
Need clarification for that step:

> Click somewhere else. Make it go away.

Click where? Inside the popup panel or outside the popup panel?
Make what go away? The popup panel?

**Comment by lasers** [comment]
Outisde the popup panel. Make UBO's popup panel go away.

**Comment by gorhill** [comment]
Ok so I can't reproduce then. At step 5, when I hit the pause button, the reload icon appears properly.

**Comment by lasers** [comment]
Sorry. I need to clarify things some more. The reload icon will appear properly... but not for correct event (on/off) that I wanted to perform so I have to redo some steps to get the next event.

**Comment by gorhill** [comment]
> The reload icon will appear properly... but not for correct event (on/off) that I wanted to perform

As said, I can't reproduce, when I disable uBO (power button grayed) at step 5, the reload icon appear.

**Comment by lasers** [comment]
1. UBO ON. Visit the toolbar icon.
2. UBO ON: Hit the big blue pause button.
  _(Blue is now gray. Reload button appears.)_

3. UBO OFF: Make pop panel go away.
   _(With this time, I copied some input forms with clipboard manager
   because I don't want to lose them... Just in case.)_

I want to reload the page. Attempting...

4. UBO OFF. Visit the toolbar icon again.
5. UBO OFF: See the big gray pause button. No button.
   _(This is when I realized that it does not know that the page hadn't been reloaded yet.)_

I still want to reload the page. Attempting...

6. UBO ON: Click the big gray pause button.
   _(Gray is now blue. This is not what I wanted.)_

7. Keep pushing the button: changes colors and make reload button go on/off.
   _(It does not work / stay still this way.)_

Multiple Solutions

A: Visit the icon toolbar. Hit the big pause button.
   Make the pop panel go away. Visit the icon toolbar again. Hit the big pause button. Reload.

B: Remember to do it via builtin browser instead.
C: Remember to use the reload shortcut instead.

Maybe it's a possible minor bug or possibly not a minor bug.
